exporter/sapmexporter/README.md

```diff
@@ -0,0 +1,40 @@
+# SAPM Exporter
+
+SAPM exports builds on the Jaeger proto and adds additional batching on top. This allows
+the collector to export traces from multiples nodes/services in a single batch. SAPM proto
+and some useful related utilities can be found [here](https://github.com/signalfx/sapm-proto/)
+
+## Configuration
+
+Example:
+
+```yaml
+exporters:
+  sapm:
+    endpoint: test-endpoint
+    access_token: abcd1234
+    num_workers: 8
+    max_connections: 100
+```
+
+* `endpoint`: This is the destination to where traces will be sent to in SAPM format. It must be a full URL and include the scheme, port and path e.g, https://ingest.signalfx.com/v2/trace. This can be pointed to the SignalFx backend or to another Otel collector that has the SAPM receiver enabled. Has no default value.
```

Comment:
nit: similar on few other places below
```suggestion
* `endpoint`: This is the destination to where traces will be sent to in SAPM format. It must be a full URL and include the scheme, port and path e.g, https://ingest.signalfx.com/v2/trace. This can be pointed to the SignalFx backend or to another Otel collector that has the SAPM receiver enabled. It has no default value.
```